src/App/Tracker/Controller/Hooks/ReceiveCommentsHook.php

```diff
@@ -143,11 +154,16 @@ protected function insertComment()
 		}
 		catch (\Exception $e)
 		{
-			$this->logger->error(
-				'Error loading the database for comment '
-				. $this->hookData->issue->number
-				. ':' . $e->getMessage()
+			$logMessage = sprintf(
+				'Error loading the database for comment %d:%s',
+				$this->hookData->issue->number,
+				$e->getMessage()
 			);
+			$this->logger->error($logMessage);
+			$this->setStatusCode($e->getCode());
+			$this->response->error = $logMessage;
+
+			return false;
```

Comment:
I'd say no as this doesn't cause a critical error.  We should send non-200 responses only when there's a critical fail (i.e. we couldn't parse the hook, there was a failure inserting the hook's issue/comment into the database, etc.) but let the hook be seen as successfully completing if the only fail comes from optional event listeners.